app/models/tenant.rb

```diff
@@ -286,9 +286,11 @@ def self.seed
   #     [["tenant/tenant2/project4", 4]]
   #   ]
   def self.tenant_and_project_names
-    tenants_and_projects = Tenant.in_my_region.select(:id, :ancestry, :divisible, :use_config_for_attributes, :name)
+    all_tenants_and_projects = Tenant.in_my_region.select(:id, :ancestry, :divisible, :use_config_for_attributes, :name)
+    tenants_by_id = all_tenants_and_projects.index_by(&:id)
+
+    tenants_and_projects = Rbac.filtered(Tenant.in_my_region.select(:id, :ancestry, :divisible, :use_config_for_attributes, :name))
```

Comment:
I think it is better to pass the local variable into rbac (could you verify it is not worse?):

```ruby
tenants_and_projects = Rbac.filtered(all_tenants_and_projects)
```

bummed about the double query here, but looks necessary